### PR TITLE
Add trim_amount()

### PR DESCRIPTION
**Pull request by EmilHvitfeldt**
This is needed to make https://github.com/tidymodels/recipes/pull/1101 pass its tests.

### CODE REVIEW

File: R/trim_amount.R [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+#' @examples
+#' trim_amount()
+#' @export
+trim_amount <- function(range = c(0, 0.5), trans = NULL) {
+  new_quant_param(
+    type = "double",
+    range = range,
+    inclusive = c(TRUE, TRUE),
+    trans = trans,
+    label = c(trim_amount = "Kernel trim_amount"),
```
Comment (hfrick) [inline R/trim_amount.R:15]: It's not immediately obvious to me what this has to do with kernels. Is that intentional or some c+p left over? The labels generally try to explain what the parameter is, without repeating the name. Could you come up with a label that tries to include these two aspects?

**Review by hfrick — COMMENTED** [review]
Looks good! Just one comment about the label

File: R/trim_amount.R (hunk 1 above)
Comment (EmilHvitfeldt) [reply R/trim_amount.R:15]: copy paste error....

**Comment by EmilHvitfeldt** [comment]
It should be fixed now!

**Pull request merged**